### DISCUSSION TITLE
zh/blog/gomoku

### DISCUSSION THREAD

**Discussion by giscus** [selfboot/ai_gallery#7] [Q&A]
# zh/blog/gomoku

用 Claude3.5 实现一个完整的单机五子棋游戏，支持各种禁手规则。如何更好的利用 AI 解决复杂问题，并让 AI 写测试用例来保证逻辑的正确性，本文会给你一个答案。

https://gallery.selfboot.cn/zh/blog/gomoku

**Comment by RealPeopleOfChina** [comment]
你能写一个ai玩贪食蛇的游戏吗？并且ai要强到无敌。

**Reply by selfboot** [reply]
贪吃蛇说实话 AI 没啥难度，等有空了写个。 不过最近在写五子棋的 ai，还是有点复杂的
